firmware/Core/Inc/antenna_deploy_drivers/ant_types.h

```diff
@@ -0,0 +1,8 @@
+#ifndef __INCLUDE_GUARD_ANT_TYPES_H__
+#define __INCLUDE_GUARD_ANT_TYPES_H__
+
+#include <stdint.h>
```

Comment:
Awesome you created this file, but it doesn't look like it ended up getting used (because the subsystem is so simple). Please delete it.

Reply:
Removed file.